Source/ACE/Entity/Generator.cs

```diff
@@ -62,11 +62,11 @@ public bool CorpseGeneratedRare
             set { AceObject.CorpseGeneratedRare = value; }
         }
 
-        public bool SuppressGenerateEffect
-        {
-            get { return AceObject.SuppressGenerateEffect ?? false; }
-            set { AceObject.SuppressGenerateEffect = value; }
-        }
+        ////public bool SuppressGenerateEffect
```

Comment:
Either keep or delete this code.  Block comments in general are unclean and will lead to an unclean codebase.